benchmark/common.js

```diff
@@ -43,7 +43,7 @@ Benchmark.prototype._parseArgs = function(argv, configs) {
   // Parse configuration arguments
   for (const arg of argv) {
     const match = arg.match(/^(.+?)=([\s\S]*)$/);
-    if (!match || !match[1]) {
+    if (!match || !match[2]) {
```

Comment:
I think you’re right, but it might be even simpler to change `*` to `+` in the regex (unless I’m missing something)?